pi5/board_test.go

```diff
@@ -33,24 +33,13 @@
 // 	b = &Board{
 // 		Named:         board.Named("foo").AsNamed(),
 // 		gpioMappings:  nil,
-// 		analogReaders: map[string]*wrappedAnalogReader{"an": {}},
 // 		logger:        logging.NewTestLogger(t),
 // 		cancelCtx:     ctx,
 // 		cancelFunc: func() {
 // 		},
 // 	}
 
-// 	t.Run("test analog-readers digital-interrupts and gpio names", func(t *testing.T) {
-// 		ans := b.AnalogNames()
-// 		test.That(t, ans, test.ShouldResemble, []string{"an"})
-
-// 		an1, err := b.AnalogByName("an")
-// 		test.That(t, an1, test.ShouldHaveSameTypeAs, &wrappedAnalogReader{})
-// 		test.That(t, err, test.ShouldBeNil)
-
-// 		an2, err := b.AnalogByName("missing")
-// 		test.That(t, an2, test.ShouldBeNil)
-// 		test.That(t, err, test.ShouldNotBeNil)
+// 	t.Run("test digital-interrupts and gpio names", func(t *testing.T) {
```

Comment:
@mariapatni Is there a reason why all the tests are commented out? Can I uncomment them?

Reply:
no reason - I just uncommented out everything I wasn't using at the time when I was creating the module

Reply:
gotcha - going to just delete this file and we can add a test file again when we add actual unit tests